### DISCUSSION TITLE
Its posible to use only GPIO26 in analog mode ?

### DISCUSSION THREAD

**Discussion by PowerTcX** [pschatzmann/arduino-audio-tools#486] [Q&A] [answered]
Its posible to leave only GPIO26 in analog mode ?
Im using Sunton 3.5* Display board, and GPIO25 is connected to touch driver rst pin, so i need only GPIO26 as output.
Im found some options in AnalogAudio.h, but no variant to use only one channel.
If leave as is - i get very disorted sound, almost inaudible.

**Comment by pschatzmann** [comment, marked as answer]
I honestly don't know and would tend to think rather not: Here is the info that I was using to implement the functionaliy:
https://docs.espressif.com/projects/esp-idf/en/v4.4.3/esp32/api-reference/peripherals/i2s.html
